README.md

```diff
@@ -246,6 +262,28 @@ kubectl scale deployment.apps/coredns -n kube-system --replicas=0
 kubectl scale deployment.apps/coredns -n kube-system --replicas=20
 ```
 
+## Github Docker Repository
+
+We use Github repository (GHCR) to keep Docker images used by the templates in 
+this package which are automatically created on each push to main using Github 
+Actions. GHCR [requires a token](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) 
+to be able to pull images. 
+
+The template expects the GHCR token to be stored in `config/ghcr_auth.json` and
+be of the following form:
+
+```
+{"auths":{"ghcr.io":{"auth":"<USERNAME + TOKEN>"}}}
+``
+
+To generate that value you will need to concatenate your username with your 
+token and base64 the results:
+
+```
+echo -n "<username>:<token>" | base64
```

Comment:
I would advise against using it on shared hosts as the token will appear in ~/.bash_history. 
Maybe setting `HISTCONTROL=ignorespace` and adding a space in front of `echo` helps a bit, but I would still be nervous running it on anything that is not my personal device

Reply:
You don't have to run it on this shared computer, I just gave an example option. So that's clearer

Reply:
The docker creation already happens in a GitHub action. This token is needed to pull the image (gh doesn't let you pull without one).

Reply:
> You don't have to run it on this shared computer, I just gave an example option. So that's clearer

What makes me nervous about this example is that it does not mention this problem, so anyone copy-pasting commands will run into this issue. So I'd rather remove this example at all if there is no good solution and just ask to use authenticated session (pointing to GH doc) or find a way how to make it safe